.github/workflows/linter-pyright.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
         pip install -r requirements-dev.txt
-    - uses: jakebailey/pyright-action@v1
+    - uses: jakebailey/pyright-action@v2
```

Comment:
No action from Microsoft / GitHub?

I see this action here: https://github.com/jakebailey/pyright-action

Maybe **use a precise commit hash** for all actions not maintained by big organizations?

Otherwise, a little bit of vulnerability to supply chain attacks: the tag is mutable, an action can make use of publishing rights.

Reply:
Done, now using a commit hash.